lib/Test/Stream/Workflow/Task.pm

```diff
@@ -114,11 +114,16 @@ sub run {
 
     my $unit = $self->{+UNIT};
     my $ctx = $unit->context;
+    my $meta = $unit->meta;
+
+    my $skip;
+    $skip = $meta->{skip} if $meta && defined $meta->{skip};
+    $skip ||= $ctx->debug->_skip; # Private accessor for deprecated thing, just until it goes away
```

Comment:
Probably written better as:
```perl
my $skip = ( $meta && defined $meta->{skip} ) 
   ? $meta->{skip} 
   : $ctx->debug->_skip;
```

Unless of course you wanted `$meta->{skip} == 0` to also result in `$ctx->debug->_skip` . 

But if that was the case, then you could have simply done:

```perl
my $skip = ( $meta && $meta->{skip} ) 
   ? $meta->{skip} 
   : $ctx->debug->_skip;
```